pkgs/development/libraries/spice/default.nix

```diff
@@ -51,11 +56,8 @@ stdenv.mkDerivation rec {
     install ${doxygen_sh} doxygen.sh
     patchShebangs build-aux
 
-    # https://gitlab.freedesktop.org/spice/spice-common/-/issues/5
-    substituteInPlace subprojects/spice-common/meson.build \
-      --replace \
-      "cmd = run_command(python, '-m', module)" \
-      "cmd = run_command(python, '-c', 'import @0@'.format(module))"
+    # Why is this up to the packager?
```

Comment:
Probably because we remove the `.git` directory.

Reply:
I think they have special release scripts where they put things like this in the tarball. I'd rather they simply record the current version somewhere in the repo and bump that on every release.